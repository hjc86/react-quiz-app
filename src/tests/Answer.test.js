// import React, { Component } from 'react';
// import ReactDOM from 'react-dom'
// import { shallow, mount } from 'enzyme';
// import Answer from '../components/Answer';
// import renderer from 'react-test-renderer';

// describe("<Answer />", () => {
    
//     /// setup for test ///

//     const essentialProps ={
//         scoreArray:[0,1],
//         userName:['a','b']
//     }

//     let componentWithProps = <Answer {...essentialProps}/>
//     let wrapper;

//     /// end of setup //

//     beforeEach(() => wrapper = shallow(componentWithProps));

//     it('should render a single <div />',()=>{
//         expect(wrapper.find('div').length).toEqual(2);
//     })

//     it('mounts to DOM', () => {
//         const div = document.createElement('div');
//         ReactDOM.render(componentWithProps, div);
//         ReactDOM.unmountComponentAtNode(div);
//     });

//     it('renders correctly', () => {
//         const tree = renderer
//             .create(componentWithProps)
//             .toJSON();
//         expect(tree).toMatchSnapshot();
//     });
    
//     it('Renders correctly', () => {
//         expect(wrapper).toMatchSnapshot();
//     })


// })
