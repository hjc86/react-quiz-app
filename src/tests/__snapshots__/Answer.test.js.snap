// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Answer /> Renders correctly 1`] = `ShallowWrapper {}`;

exports[`<Answer /> renders correctly 1`] = `
<div
  className="buttonDiv"
>
  <button
    className="answers undefined"
    onClick={[Function]}
    style={
      Object {
        "background": "#ffdd59",
        "display": undefined,
      }
    }
  />
</div>
`;
